pack.ps1

```diff
@@ -25,6 +27,15 @@ function createNuGetPackage([string]$pkgName, [string]$culture)
     $cultureFolder = [IO.Path]::Combine($localizationFolderName, $culture)
     Copy-Item -Path $cultureFolder -Destination $pkgCultureFolderPath -Recurse
     
+    echo "Copying '.props' template file .."
+
+    $pkgBuildFolderPath = [IO.Path]::Combine($pkgFolderPath, "buildTransitive")
```

Comment:
Can we introduce a variable that hold "buildTransitive" like what we did for all the folders

Reply:
Okay, i did it like that because you did the same before for `content`

Reply:
done

Reply:
It's fine, because it's used once .. just leave it as it is